orthanc/orthanc-anon/plugin/pixl.py

```diff
@@ -237,7 +237,11 @@ def OnChange(changeType, level, resource):  # noqa: ARG001
         logger.info(msg)
         SendViaFTPS(resource)
 
-    if changeType == orthanc.ChangeType.ORTHANC_STARTED and _azure_available():
+    if (
+        changeType == orthanc.ChangeType.ORTHANC_STARTED
+        and _azure_available()
+        and ShouldAutoRoute()
```

Comment:
Since we have an early return if `ShouldAutoRoute()` is false in https://github.com/UCLH-Foundry/PIXL/blob/9b17f44b473d9ad970e11af8704c72fb290a53b7/orthanc/orthanc-anon/plugin/pixl.py#L232, we don't actually need this check here I think? Same goes for line 235.

Reply:
ah yeah good call, if only I could read